lib/msal-core/samples/react-sample-app/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./App.css";
+
+import AuthProvider from "./AuthProvider";
+
+class App extends React.Component {
+    render() {
+        return (
+            <div>
+                <section>
+                    <p>
+                        Welcome to the Microsoft Authentication Library For
+                        Javascript - React Quickstart
+                    </p>
+                    {!this.props.account ? (
+                        <button onClick={this.props.signIn}>Sign In</button>
+                    ) : (
+                        <>
+                            <button onClick={this.props.signOut}>
+                                Sign Out
+                            </button>
+                            <button onClick={this.props.readMail}>
+                                Read Email
+                            </button>
+                        </>
+                    )}
+                </section>
+                <section>
+                    {this.props.account && (
```

Comment:
Should we add headers to the pre blocks? 
So its clear what the json data represents